gatsby-ssr.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import type { GatsbySSR } from 'gatsby';
+
+export const onRenderBody: GatsbySSR['onRenderBody'] = ({ setHeadComponents }) => {
+  setHeadComponents([
+    <link
+      rel="preload"
+      href="/fonts/Druk-Super-Web.woff2"
+      as="font"
+      type="font/woff2"
+      crossOrigin="anonymous"
+      key="druk-super-woff2"
+    />,
+    <link
+      rel="preload"
+      href="/fonts/DrukWide-Bold-Web.woff2"
+      as="font"
+      type="font/woff2"
+      crossOrigin="anonymous"
+      key="drukwide-bold-woff2"
+    />,
+  ]);
+};
```

Comment:
Using the TS format of `gatsby-ssr.tsx` file
https://www.gatsbyjs.com/docs/how-to/custom-configuration/typescript/#gatsby-browsertsx--gatsby-ssrtsx

Also, following this method of adding local fonts here:
https://www.gatsbyjs.com/docs/how-to/styling/using-local-fonts/